docs/src/routes/docs/components/dot.svx

```diff
@@ -0,0 +1,86 @@
+---
+name: Dot
+---
+
+<script>
+  import { Dot, Button } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+</script>
+
+# Dot
+
+A dot.
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <Button>
+      Plain ol' dot: <Dot />
+    </Button>
+    <Button>
+      New info <Dot info class="ml" />
+    </Button>
+    <Button>
+      <Dot success class="mr" />Mission accomplished
+    </Button>
+    <Button>
+      Take care! <Dot attention class="ml" />
+    </Button>
+    <Button>
+      Small warning <Dot danger small class="ml" />
+    </Button>
+  </div>
+  <div slot="source">
+
+```svelte
+<Button>
+  Plain ol' dot: <Dot />
+</Button>
+<Button>
+  New info <Dot info class="ml" />
+</Button>
+<Button>
+  <Dot success class="mr" />Mission accomplished
+</Button>
+<Button>
+  Take care! <Dot attention class="ml" />
+</Button>
+<Button>
+  Small warning <Dot danger small class="ml" />
+</Button>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`info`** | `false` | `boolean` | Applies the `$info` color to the dot |
+| **`attention`** | `false` | `boolean` | Applies the `$attention` color to the dot |
+| **`danger`** | `false` | `boolean` | Applies the `$danger` color to the dot |
+| **`success`** | `false` | `boolean` | Applies the `$success` color to the dot |
```

Comment:
We should probably mention that all these color props are mutually exclusive, kind of like it's done in the button. Listing them all in each cell would be too much and too repetitive, so how about we write this in a paragraph right above the table? Don't forget the `==`, to remain consistent with the Button, when listing those props in that paragraph